packages/dashboard/src/components/deliveriesTable.tsx

```diff
@@ -48,6 +68,48 @@ interface DeliveriesState {
   paginationRequest: EphemeralRequestStatus<Error>;
 }
 
+interface PreviewObjectInterface {
+  subject: string | null | undefined;
+  from: string | null | undefined;
+  replyTo: string | null | undefined;
+  to: string;
+  body: string | null;
+  show: boolean;
+  channelType: string;
+}
+
+const initPreviewObject = () => {
+  return {
+    subject: "",
+    from: "",
+    replyTo: "",
+    to: "",
+    body: "",
+    show: false,
+    channelType: "",
+  };
+};
+
+const BodyBox = styled(Box, {
```

Comment:
I believe you've copy and pasted this code. Maybe you can pull it out into a component to share instead?